astropy/visualization/wcsaxes/grid_paths.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 from matplotlib.lines import Path
 
-from astropy.coordinates.angle_utilities import angular_separation
+from astropy.coordinates.angles import angular_separation
```

Comment:
Just in case you didn't get it already in your search&replace, this can also be `astropy.coordinates` itself.

Reply:
Done.